packages/header-checker-lint/test/header-checker-lint.ts

```diff
@@ -39,15 +43,30 @@ describe('HeaderCheckerLint', () => {
         }),
       },
     });
-
     probot.load(myProbotApp);
   });
 
   describe('opened pull request', () => {
     let payload: EventPayloads.WebhookPayloadPullRequest;
+    const sandbox = sinon.createSandbox();
+    let getConfigStub: sinon.SinonStub;
+    let validateConfigStub: sinon.SinonStub;
 
     beforeEach(() => {
       payload = require(resolve(fixturesPath, './pull_request_opened'));
+      getConfigStub = sandbox.stub(botConfigUtilsModule, 'getConfig');
+      // By default, getConfigStub resolves to null.
+      getConfigStub.resolves(null);
```

Comment:
nit: I might leave this unset, so we don't need to call `reset()` in all the tests that do use config. Then we would explicitly resolve `null` in the test cases that don't have config.

Reply:
sounds good